proxy.conf.json

```diff
@@ -19,6 +19,15 @@
         "secure": false,
         "logLevel" : "debug"
     },
+    "/delfin" : {
+        "target": "http://192.168.56.110:8190",
```

Comment:
is this a sample deafult cof file ? target can be 127.0.0.0 ??

Reply:
This is local development prox conf. The IP address has to be replaced with the local address. This info is in the development branch Readme.